### ISSUE TITLE
TES inspection example fails with DaVinci > v36r6

### ISSUE THREAD

**Issue by betatim** [lhcb/first-analysis-steps#147]
``` python
import sys

from GaudiConf import IOHelper
from Configurables import DaVinci
import GaudiPython as GP

input_files = [sys.argv[-1]]
IOHelper('ROOT').inputFiles(input_files)

dv = DaVinci()
dv.DataType = '2012'

app_mgr = GP.AppMgr()
evt_svc = app_mgr.evtsvc()

app_mgr.run(1)
evt_svc.dump()
```

this script fails with versions of DaVinci > v36r6. It goes crazy while printing the TES:

```
IncrementalExecutor::executeFunction: symbol '_ZN10GaudiUtilslsIN4LHCb10ParticleIDESaIS2_EEERSoS4_RKSt6vectorIT_T0_E' unresolved while linking function '__cxx_global_var_init192'!
You are probably missing the definition of std::ostream& GaudiUtils::operator<< <LHCb::ParticleID, std::allocator<LHCb::ParticleID> >(std::ostream&, std::vector<LHCb::ParticleID, std::allocator<LHCb::ParticleID> > const&)
```

Reported by @elgicse

**Comment by elgicse** [comment]
problem happens between the printouts of `Event/Rec/Header` and of `Event/Rec/Status`

**Comment by alexpearce** [comment]
I saw this as well. Everything still seemed to work though. Did everything still work for you?

**Comment by alexpearce** [comment]
Sorry, didn't see this was reported by @elgicse, never mind.
